packages/next-auth/src/core/lib/oauth/authorization-url.ts

```diff
@@ -7,6 +7,7 @@ import type { AuthorizationParameters } from "openid-client"
 import type { InternalOptions } from "../../types"
 import type { RequestInternal } from "../.."
 import type { Cookie } from "../cookie"
+import { createNonce } from "./nonce-handler"
```

Comment:
(I'm not a maintener but I really hope this PR to be merged 😊 Thank you!)

I think `import { createNonce } from "./nonce-handler"` shoud be placed under `import { createPKCE } from "./pkce-handler"` (line 4)
(and  `import {useNonce} ..` in `callback.ts` should also be placed after `import { usePKCECodeVerifier } from "./pkce-handler"`)

Reply:
@ThangHuuVu  - Happy to move these as per above feedback, but note the linter doesn't seem to have any preference on ordering the imports. @ThangHuuVu - What is your opinion?

Reply:
Yep, let's do it as @kesoji suggested! 🙌

Reply:
@ThangHuuVu this has now been addressed